mobile/android/logcat-helper.ts

```diff
@@ -10,28 +10,28 @@ export class LogcatHelper implements Mobile.ILogcatHelper {
 			private $logger: ILogger,
 			private $staticConfig: Config.IStaticConfig) { }
 
-	public start(deviceIdentifier: string): any {
-		let adbPath = this.$staticConfig.getAdbFilePath().wait();
-		this.$childProcess.exec(util.format("%s -s %s logcat -c", adbPath, deviceIdentifier)).wait(); // remove cached logs
-		let adbLogcat = this.$childProcess.spawn(adbPath, ["-s", deviceIdentifier, "logcat"]);
-		let lineStream = byline(adbLogcat.stdout);
+	public start(deviceIdentifier: string): void {
+		if(deviceIdentifier) {
+			let adbPath = this.$staticConfig.getAdbFilePath().wait();
+			this.$childProcess.exec(util.format("%s -s %s logcat -c", adbPath, deviceIdentifier)).wait(); // remove cached logs
```

Comment:
this will be handled in a separate PR (related to all changes I'm making in NativeScript CLI)